sdk/identity/perf-tests/identity/README.md

```diff
@@ -0,0 +1,13 @@
+### Guide
+
+**Important:**  
+These tests won't work on Node 8 nor Node 15.
+
+1. Install `@azure/msal-node-extensions@1.0.0-alpha.6` globally with `npm i -g @azure/msal-node-extensions@1.0.0-alpha.6`.
```

Comment:
Why global.. Is it not enough to add it as a dependency in the package.json?

Reply:
We can't have this dependency in our package.json since this means developers need to have Visual Studio or build-essential installed, and in some Node versions this dependency doesn't compile. We're working on it with the MSAL team.

Reply:
I meant.. dependency of the perf-test identity project, where you're mentioning `@azure/identity`.